mobile/lib/modules/backup/models/hive_backup_albums.model.dart

```diff
@@ -46,6 +46,15 @@ class HiveBackupAlbums {
     );
   }
 
+  HiveBackupAlbums deepCopy() {
```

Comment:
What purpose will deepCopy will be served?

Reply:
It allows to get a copy that is actually safe to modify without changing the global state of `HiveBackupAlbums`.

Hive does not work like it's API indicates that it works. When you do:

1.  `info = box.get(backupInfoKey);`
2.  modify a few entries in `info`
3. ignore `info` and do nothing with it anymore
4. in some other method load `otherInfo = Hive.box<HiveBackupAlbums>(hiveBackupInfoBox).get(backupInfoKey)`

You would expect `info != otherInfo`. However, the changes made to `info` in 2. are also visible in `otherInfo` although it was not saved/put back into the box!

To guard against this, a deep copy of the arrays in `HiveBackupAlbums` is required.